Cargo.toml

```diff
@@ -6,7 +6,7 @@ version = "1.0.0-beta.7"
 authors = ["Tobie Morgan Hitchcock <tobie@surrealdb.com>"]
 
 [features]
-default = []
+default = ["surrealdb/kv-rocksdb", "surrealdb/scripting", "surrealdb/http"]
```

Comment:
Hey @rushmorem , could we make these separate features, just to keep it consistent...

```rs
default = ["storage-rocksdb", "scripting", "http"]
storage-rocksdb = ["surrealdb/kv-rocksdb"]
storage-tikv = ["surrealdb/kv-tikv"]
storage-fdb = ["surrealdb/kv-fdb"]
scripting = ["surrealdb/scripting"]
http = ["surrealdb/http"]
```